exasol/ds/sandbox/lib/s3/buckets.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+from exasol.ds.sandbox.lib.aws_access.aws_access import AwsAccess
+from exasol.ds.sandbox.lib.config import ConfigObject
+from exasol.ds.sandbox.lib.logging import get_status_logger, LogType
+from exasol.ds.sandbox.lib.asset_id import AssetId
+from exasol.ds.sandbox.lib.aws_access.cloudformation_template import CfStackSpec, CfTemplate
+
+
+LOG = get_status_logger(LogType.S3_BUCKETS)
+
+VM_BUCKET = CfStackSpec(
+    cf_stack_name="DATA-SCIENCE-SANDBOX-VM-Bucket",
+    template="vm_bucket_cloudformation.jinja.yaml",
+    outputs={
+        "BucketId": "VMBucketId",
+        "ExportRoleId": "VMExportRoleId",
+        "CfDistributionId": "CfDistributionId",
+        "CfDistributionDomainName": "CfDistributionDomainName",
+    },
+)
+
+
+class S3Bucket(CfTemplate):
+    def __init__(self, aws_access: Optional[AwsAccess], spec: CfStackSpec):
+        super().__init__(aws_access, spec)
+
+    @classmethod
+    def vm(cls, aws_access: Optional[AwsAccess]) -> "S3Bucket":
```

Comment:
Why `vm`?

Reply:
Because this is the CF stack for the S3 Bucket that is designated to contain the VM images (virtual machine) of the ai-lab. 
I am happy to search for better names but I'd love to also find short names.

Reply:
See next push.